docs/actions.md

```diff
@@ -1,182 +1,129 @@
 # Actions
 
-Use [actions](/docs/api.md#actions) to update the state.
+Actions are functions which take the current [state](/docs/state.md) and return a partial state or a [thunk](#thunks). Actions represent the intent to manipulate the state tree.
```

Comment:
Is this link to thunks correct? Or should it be [Thunks](/docs/actions.md#thunks)